test/benchmark.py

```diff
@@ -62,15 +62,32 @@ def human_int(value):
     return int(value) * multiplier
 
 
-def create_test_file(path, max_size, verbose=False):
+def create_test_file(path, max_size, verbose=False, chunk_size=65535):
```

Comment:
We should use a power of 2 for the chunk_size.

```suggestion
def create_test_file(path, max_size, verbose=False, chunk_size=65536):
```